ovos_gui_plugin_shell_companion/brightness.py

```diff
@@ -260,12 +260,10 @@ def start_auto_night_mode(self, message=None):
                                                 name="ovos-shell.night.mode.check")
             if self.sunset_time < date < self.sunrise_time:
                 LOG.debug("It is night time")
-                self.bus.emit(Message("phal.brightness.control.auto.night.mode.enabled"))
+                self.bus.emit(Message("speaker.extension.display.set.auto.nightmode", {"auto_nightmode": True}))
             else:
                 LOG.debug("It is day time")
-                # TODO - implement this message in shell / check if it exists
-                # i just made it up without checking
-                self.bus.emit(Message("phal.brightness.control.auto.night.mode.disabled"))
+                self.bus.emit(Message("speaker.extension.display.set.auto.nightmode", {"auto_nightmode": True}))
```

Comment:
Should this be `False` to disable nightmode?

Reply:
Good catch...I wrote this patch for another user who initially reported it was good, but I haven't tested myself. I installed this on my office Mark 2 and I'll report back in the morning. Right now night mode is enabled but that seems to never turn off when I turn it on...

Reply:
that was the bug this is supposed to fix, it never comes out of night mode when it turns into daytime

Reply:
I'll let you know in the morning. ;)